static/app/components/devtoolbar/components/feedback/feedbackPanel.tsx

```diff
@@ -49,7 +50,20 @@ export default function FeedbackPanel() {
       [setVisible]
     )
   );
-  const transactionName = useCurrentTransactionName();
+
+  const {scope, client} = useScopeAndClient();
+  const [transactionName, setTransactionName] = useState(
+    scope?.getScopeData().transactionName
+  );
+
+  useEffect(() => {
+    if (client) {
+      client.on('startNavigationSpan', options => {
+        setTransactionName(toSearchTerm(options.name));
+      });
+    }
+  }, [client]);
```

Comment:
We should return a cleanup fn that cleans up the listener

Reply:
is the listener the return value of client.on?

Reply:
how do you clean up the listener?